src/backend/distributed/planner/deparse_shard_query.c

```diff
@@ -350,6 +335,82 @@ UpdateRelationsToLocalShardTables(Node *node, List *relationShardList)
 }
 
 
+static bool
+ReplaceRelationConstraintByShardConstraint(Oid constraintId, List *relationShardList,
```

Comment:
The convention is to have a comment on functions (I am kind of not a supporter for this convention but still it is a convention :))